.pipelines/rec_dtdl_prod.yml

```diff
@@ -0,0 +1,42 @@
+name: Ontology Push To Nuget
+
+on:
+  workflow_dispatch:    
+  push:
+    branches: [ "main" ]
+    paths:
+      - "Ontology"
+
+env: 
+  NUGET_FEED_URL: https://api.nuget.org/v3/index.json
+  BUILD_CONFIGURATION: 'Release'    # set this to the appropriate build configuration
+  DOTNET_VERSION: '6.x' 
+  NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }} 
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repo
+      - uses: actions/checkout@v3
+      - name: 'Setup Nuget'
+        uses: nuget/setup-nuget@v1
+        with:
+          nuget-api-key: ${{ secrets.NuGetAPIKey }}
+          nuget-version: '5.x'
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v2
+        with:
+          dotnet-version: 6.0.x
+
+      # Run nuget package
+      - name: 'nuget pack'
+        run: |
+          nuget pack Ontology/Ontology.nuspec
+
+      # Publish the Microsoft.SmartPlaces.Facilities.OntologyMapper.Mapped package to Nuget
+      - name: 'dotnet publish to nuget'
+        run: dotnet nuget push --source "${{ env.NUGET_FEED_URL }}" Ontology/**/*.nupkg
```

Comment:
Should the skip duplicates flag be added to avoid failed builds since the trigger for the build was defined? Right now there is only one nuget, so a failed build makes sense, but if there are multiple nugets (wildcard indicates this possibility)  unless they're in lockstep a failed build is rather misleading.